src/components/CompanyBioCard.jsx

```diff
@@ -0,0 +1,126 @@
+import styled from '@emotion/styled';
+
+const CompanyBioContainer = styled.div`
+  ${({ theme }) => `
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-color: ${theme.companyBioCard.bgColor};
+    border-radius: 6px;
+
+    @media (min-width: ${theme.breakpoints.tablet_small}) {
+      flex-direction: row;
+      height: 140px;
+    }
+  `};
+`;
+
+const LogoImageWrapper = styled.div`
+  background-color: ${({ logoBackground }) => logoBackground};
+
+  ${({ theme }) => `
+    width: 50px;
+    height: 50px;
+    display: flex;
+    border-radius: 16px;
+    align-items: center;
+    justify-content: center;
+    margin-top: -1.5rem;
```

Comment:
Instead of using a negative margin, let's use `transform: translateY(-1.5rem)` instead. This approach is generally more performant and less likely to cause layout issues.

Reply:
Great thanks, that works!